dumb-init.spec

```diff
@@ -0,0 +1,46 @@
+Name:           dumb-init
+Version:        1.1.3
+Release:        2%{?dist}
+Summary:        Entry-point for containers that proxies signals
+
+License:        MIT
+URL:            https://github.com/Yelp/dumb-init
+Source0:        https://github.com/Yelp/dumb-init/archive/v%{version}.tar.gz
+
+BuildRequires:  vim-common
```

Comment:
still curious why this depends on vim-common? (GitHub buried my original comment, it seems)